distribution/docs/src/main/resources/_contents/_catalog-frameworks/catalog-api-contents.adoc

```diff
@@ -0,0 +1,247 @@
+
+==== Catalog API
+
+The Catalog API is an OSGi bundle (`catalog-core-api`) that contains the Java interfaces for the Catalog components and implementation classes for the Catalog Framework, Operations, and Data components.
+
+===== Catalog API Search Interfaces
+
+====== ${ddf-ui} Application Search Interface
+
+The ${branding} Search UI application provides a graphic interface to return results and locate them on an interactive globe or map.
+
+====== SSH Search Interface
```

Comment:
Maybe we should just rename to something like "Using SSH to search through the command line" ?